hack/ci-e2e.sh

```diff
@@ -106,16 +107,42 @@ export IMAGE_URL="http://${IP_ADDRESS}/${IMAGE_FILE}"
 IMAGE_DIR="${REPO_ROOT}/test/e2e/images"
 mkdir -p "${IMAGE_DIR}"
 
-## Download and run image server
-wget --quiet -P "${IMAGE_DIR}"/ https://artifactory.nordix.org/artifactory/metal3/images/iso/"${IMAGE_FILE}"
+## Download disk images
+wget --quiet -P "${IMAGE_DIR}/" https://artifactory.nordix.org/artifactory/metal3/images/iso/"${IMAGE_FILE}"
+wget --quiet -P "${IMAGE_DIR}/" https://fastly-cdn.system-rescue.org/releases/11.00/systemrescue-11.00-amd64.iso
 
+## Start the image server
 docker run --name image-server-e2e -d \
   -p 80:8080 \
   -v "${IMAGE_DIR}:/usr/share/nginx/html" nginxinc/nginx-unprivileged
 
-# Generate the key pair
+# Generate ssh key pair for verifying provisioned BMHs
 ssh-keygen -t ed25519 -f "${IMAGE_DIR}/ssh_testkey" -q -N ""
 
+# Build an ISO image with baked ssh key
+# See https://www.system-rescue.org/scripts/sysrescue-customize/
+# We use the systemrescue ISO and their script for customizing it.
```

Comment:
I'm open to suggestions here. Should we build the ISO in a different way? Use a different image to start from?

Reply:
In the Ironic CI, we use http://tinycorelinux.net/15.x/x86/release/Core-current.iso, but I don't know how to embed the SSH key there. If sysrescue does not take ages to build, it could be an option.